visualization/app/codeCharta/ui/screenshotButton/screenshotButton.component.ts

```diff
@@ -41,6 +66,25 @@ export class ScreenshotButtonController {
 	}
 
 	private loadScript(link: HTMLAnchorElement, renderer: WebGLRenderer) {
+		this.buildScreenShotCanvas(renderer)
+
+		link.href = renderer.domElement.toDataURL()
+		renderer.setPixelRatio(1)
+	}
+
+	makeScreenshotToClipBoard() {
+		const renderer = this.threeRendererService.renderer
+		this.buildScreenShotCanvas(renderer)
+
+		renderer.domElement.toBlob(async function (blob) {
+			const clipboardItem = new ClipboardItem({ [blob.type]: blob })
+			// @ts-ignore
+			navigator.clipboard.write([clipboardItem])
+		})
+		renderer.setPixelRatio(1)
```

Comment:
We should add a comment, why setPixelRatio must be called.

Reply:
apparently we do not need setPixelRatio

Reply:
It depends on the device resolution, 1 is a low option for high density devices (dpi> 0 -> screenshots will be jaggy). I would actually set it to a higher value ( the screenshot will be bigger on low dpi devices though) or query the device dpi and use it as a value.